pkg/capacity/manager_test.go

```diff
@@ -421,14 +406,13 @@ func TestGetOptimalTargetClusterInBatchesBy50WhenTwoClusterHaveTheSameUsage(t *t
 						WithMember("member3", WithSpaceCount(numberOfSpaces), WithNodeRoleUsage("worker", 55), WithNodeRoleUsage("master", 50)))
 
 					// member2 and member3 have the same capacity left and the member1 is full, so no one can be provisioned there
-					toolchainConfig := commonconfig.NewToolchainConfigObjWithReset(t,
-						testconfig.CapacityThresholds().
-							MaxNumberOfSpaces(testconfig.PerMemberCluster("member2", limit), testconfig.PerMemberCluster("member3", limit)))
+					spc1 := spc.NewEnabledValidTenantSPC("member1Spc", commontest.HostOperatorNs, "member1", spc.MaxNumberOfSpaces(1000))
```

Comment:
let's have the first one without any limit
```suggestion
					spc1 := spc.NewEnabledValidTenantSPC("member1Spc", commontest.HostOperatorNs, "member1")
```